tests/test_po.py

```diff
@@ -0,0 +1,210 @@
+# Copyright Mozilla Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from importlib.resources import files
+from textwrap import dedent
+from unittest import TestCase
+
+from moz.l10n.po import po_parse, po_serialize
+from moz.l10n.resource import Entry, Metadata, Resource, Section
+
+# Show full diff in self.assertEqual. https://stackoverflow.com/a/61345284
+# __import__("sys").modules["unittest.util"]._MAX_LENGTH = 999999999
+
+
+class TestPo(TestCase):
```

Comment:
This file would be easier to read if assertions would be split into separate methods.

A comment explaining what each assertion does would also help.

Reply:
Split done.